### PR TITLE
fix: properly return a failure code if any of the unit tests are unsu…

### PR DESCRIPTION
**Pull request by RedFox20**
…ccessful, but still make sure to run all the tests

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -74,17 +75,28 @@ for pool in $pools; do
...
+  wait $pid || let "FAIL+=1"
+done
+
 end_time=$(date +%s)
 elapsed=$(( end_time - start_time ))
 echo ""
 echo elapsed time: $(date -ud "@$elapsed" +'%M min %S sec')
+
+if [ "$FAIL" != "0" ]; then
+  exit $FAIL
```
Comment (yuval-weiss) [inline run_tests.sh:101]: We should probably just `exit 1` here....

Comment (yuval-weiss) [reply run_tests.sh:101]: some exit codes are reserved. 1 is general error

Comment (yuval-weiss) [reply run_tests.sh:101]: https://tldp.org/LDP/abs/html/exitcodes.html

Comment (RedFox20) [reply run_tests.sh]: Ok, I changed it to `1` and to report how many of the runners failed @yuval-weiss

**Review by mijovic — APPROVED** [review]

**Pull request merged**